### PR TITLE
Review V1 Swagger (core-command) for completeness.

### LINKED ISSUE #2359: Review V1 Swagger for Completeness -- Core-Command
All of the old RAML documents for the API endpoints in Core/Supporting services were converted to Swagger by the QA/Test team. As part of this conversion, we identified the need to review the resulting Swagger for completeness. If the original RAML document did not include comprehensive description of a service's API behavior, then obviously the converted Swagger won't either.

This issue is for review of the core-command V1 [Swagger doc](https://github.com/edgexfoundry/edgex-go/blob/master/api/openapi/v1/core-command.yaml). Please see the review @AnthonyMBonafide has already done on PR #2329 and let that inform the audit.

### PR DESCRIPTION
**Pull request by akramtexas**
Fix #2359

### CODE REVIEW

File: api/openapi/v1/core-command.yaml [hunk 1]
```diff
@@ -111,19 +105,17 @@ paths:
         200:
           description: String as returned by the device/sensor via the device service.
         400:
-          description: if the request is malformed or unparsable
+          description: If the request is malformed or unparsable
         404:
-          description: if device with given name does not exist or device doesn't
+          description: If device with given name does not exist or device doesn't
             have a command with the given commandname.
```
Comment (brandonforster) [inline api/openapi/v1/core-command.yaml]: Can we split "command name" into two words?

File: api/openapi/v1/core-command.yaml [hunk 2]
```diff
@@ -111,19 +105,17 @@ paths:
...
         423:
-          description: if the device is locked (admin state)
+          description: If the device is locked, admin state
```
Comment (brandonforster) [inline api/openapi/v1/core-command.yaml]: I think a conjunction like "in an" is more appropriate than a comma.

File: api/openapi/v1/core-command.yaml [hunk 3]
```diff
@@ -111,19 +105,17 @@ paths:
...
         500:
-          description: for unanticipated or unknown issues encountered
+          description: For unanticipated or unknown issues encountered
   /v1/device/{id}:
     get:
-      description: Retrieve a device (by database generated id) and its command offerings.
-        Throws ServiceException (HTTP 500) for unanticipated or unknown issues encountered.
-        Throws NotFoundException (HTTP 404) if no device exists by the id provided.
+      description: Retrieve a device by database generated ID and its command offerings.
```
Comment (brandonforster) [inline api/openapi/v1/core-command.yaml]: "Offerings" is kind of weird word choice.

File: api/openapi/v1/core-command.yaml [hunk 4]
```diff
@@ -169,19 +159,17 @@ paths:
...
           description: String as returned by the device/sensor via the device service.
         400:
-          description: if the request is malformed or unparsable
+          description: If the request is malformed or unparsable
         404:
-          description: if no device exists by the id provided
+          description: If no device exists by the ID provided
         423:
-          description: if the device is locked (admin state).
+          description: If the device is locked, admin state.
```
Comment (brandonforster) [inline api/openapi/v1/core-command.yaml]: Same as above. The comma is weird.

File: api/openapi/v1/core-command.yaml [hunk 5]
```diff
@@ -221,9 +213,9 @@ paths:
       description: Test service providing an indication that the service is available.
       responses:
         200:
-          description: pong as a string
+          description: Pong as a string
```
Comment (brandonforster) [inline api/openapi/v1/core-command.yaml]: Let's remove implementation detail and change this to "String literal confirming the service is up."

**Comment by akramtexas** [comment]
Rebased

**Comment by akramtexas** [comment]
> Looks good, but your build is failing.

Thanks 👍 I'm now investigating the failing build ✅

**Comment by akramtexas** [comment]
Rebased

**Pull request merged**